config/schema/webform_strawberryfield.schema.yml

```diff
@@ -0,0 +1,35 @@
+field.widget.settings.strawberryfield_webform_inline_widget:
+  type: config_object
+  label: 'Webform Inline Widget Schema'
+  mapping:
+    placeholder:
+      type: string
+      label: 'Placeholder'
+    webform_id:
+      type: string
+      label: 'Webform ID'
+    rows:
+      type: integer
+      label: 'Rows'
+field.widget.settings.strawberryfield_webform_widget:
+  type: config_object
+  label: 'Webform Inline Widget Schema'
+  mapping:
+    placeholder:
+      type: string
+      label: 'Placeholder'
+    webform_id:
+      type: string
+      label: 'Webform ID'
+    rows:
+      type: integer
+      label: 'Rows'
+# see field.widget.settings.webform_entity_reference_autocomplete
+# match_limit should already exist in schema but getting missing schema error.
+#field.widget.settings.webform_entity_reference_autocomplete:
```

Comment:
Question (have not tested yet, but maybe...who knows)
What if the type is 
`field.widget.settings.entity_reference_autocomplete` directly instead of mapping? I mean all those properties are not inherited from that main Autocomplete? We have no particular to our implementation properties right? Or do we?

That one is defined in `web/core/config/schema/core.entity.schema.yml`, so who knows, maybe right?

Reply:
@DiegoPino thanks, that suggestion helps me understand schemas more broadly. I tried that which got rid of the missing schema for match_limit, but added a missing schema error for `default_data`. So if I add both match_limit and default_data to this with the type as you suggested, no errors. Pushing that here now.

Reply:
Great!

Reply:
@DiegoPino I am going to try that approach with some of the formatter ones I had issues with. Will keep you posted over in that repo soon.